echo-sql/README.md

```diff
@@ -0,0 +1,120 @@
+# Example URL Shortener
+
+A sample url shortener app to test Keploy integration capabilities using [Echo](https://echo.labstack.com/) and PostgreSQL. 
+
+## Installation
+
+### Start keploy server
+
+```shell
+git clone https://github.com/keploy/keploy.git && cd keploy
+docker-compose up
+```
+
+### Setup URL shortener
+
+```bash
+git clone https://github.com/keploy/example-url-shortener && cd example-url-shortener/echo-sql
+go mod download
+```
+
+### Start PostgreSQL instance
+```bash
+docker-compose up -d
+```
+
+### Run the application
+
+```shell
+go run handler.go main.go
+```
+
+## Generate testcases
+
+To genereate testcases we just need to make some API calls. You can use [Postman](https://www.postman.com/), [Hoppscotch](https://hoppscotch.io/), or simply `curl`
+
+###1. Generate shortned url
+
+```bash
+curl --request POST \
+  --url http://localhost:8080/url \
+  --header 'content-type: application/json' \
+  --data '{
+  "url": "https://google.com"
+}'
+```
+
+this will return the shortened url. The ts would automatically be ignored during testing because it'll always be different.
+
+```
+{
+  "ts": 1645540022,
+  "url": "http://localhost:8080/Lhr4BWAi"
+}
+```
+
+###2. Redirect to original url from shortened url
+
+```bash
+curl --request GET \
+  --url http://localhost:8080/Lhr4BWAi
+```
+
+or by querying through the browser `http://localhost:8080/Lhr4BWAi`
+
+Now both these API calls were captured as a testcase and should be visible on the [Keploy console](http://localhost:8081/testlist).
+If you're using Keploy cloud, open [this](https://app.keploy.io/testlist).
+
+You should be seeing an app named `sample-url-shortener` with the test cases we just captured.
+
+![testcases](https://github.com/petergeorgas/samples-go/blob/main/gin-mongo/testcases.png?raw=true)
+
+Now, let's see the magic! 🪄💫
+
+## Test mode
+
+Now that we have our testcase captured, run the test file.
+
+```shell
+ go test -coverpkg=./... -covermode=atomic  ./...
+```
+
+output should look like
+
+```shell
+ok      test-app-url-shortener  6.268s  coverage: 80.3% of statements in ./...
```

Comment:
We should update this line to reflect the actual code coverage we are getting

Reply:
Code coverage has now been updated.